CoreFoundation/Base.subproj/CFPlatform.c

```diff
@@ -157,7 +157,11 @@ const char *_CFProcessPath(void) {
 
 #if DEPLOYMENT_TARGET_LINUX
 #include <unistd.h>
+#if __has_include(<syscall.h>)
 #include <syscall.h>
+#else
```

Comment:
If the previous header does not exist what is to say the sys/syscall version exists? Perhaps that should be gated as well and #error'd on the else claiming that syscall.h is required